otx/algorithms/common/adapters/torch/dataloaders/samplers/cls_incr_sampler.py

```diff
@@ -10,9 +11,12 @@
 
 from otx.algorithms.common.utils.task_adapt import unwrap_dataset
 
+# pylint: disable=too-many-instance-attributes
```

Comment:
To disable pylint as minimum as possible, how about setting in class?

Reply:
We had a consensus that we can ignore too-many-* as of now.